src/pages/p2p/components/_exchange-steps.js

```diff
@@ -154,7 +158,7 @@ const ExchangeSteps = () => {
                 </Card>
                 <Card>
                     <Row>
-                        <img src={CompleteOrder} alt="" width="100%" />
+                        <ImageTag src={CompleteOrder} alt="Complete the transaction" />
```

Comment:
We don't need alts for these icons. They serve as a visual purpose accompanying their actual text beside. No alts are needed for these as it would be duplicated and is bad for accessibility.